### DISCUSSION TITLE
A GPT for all of us

### DISCUSSION THREAD

**Discussion by kayhantolga** [betalgo/openai#503] [Announcements]
Hi all, 

 ### 🎉I have created a GPT that will help you with documentation and code. 🎉 
 This is an alpha version, so it's pretty new. I haven't tested it properly yet, but decided to share it to get feedback faster. So please give it a try and share your feedback, whether the conversations are good or poor.

Apologies for the poor naming, but due to trademark issues, I couldn't find a proper name.
👇
https://chat.openai.com/g/g-qt3kPqd3m-net-sdk-guide-by-laser-cat-eyes

**Comment by Artein** [comment]
Used to answer my question about the support of Assistants API by the SDK. Worked like a charm
